website/frontend/templates/docs/tags.html

```diff
@@ -314,6 +314,11 @@ <h1 id="basic_variables">Basic Variables</h1>
           <td>_totalalbumtracks</td>
           <td>The total number of tracks across all discs of this release.</td>
         </tr>
+        <tr>
+          <td>_musicbrainz_tracknumber</td>
+          <td>The track number as entered on MusicBrainz. This can be a non-numeric value, e.g. A and B or A1, A2 etc.
+          </td>
+        </tr>
```

Comment:
This should follow the way of script functions documentation, to have them in-app and generate this code from it instead.